adapters/mqtt-vertx-base/src/main/java/org/eclipse/hono/adapter/mqtt/MqttContext.java

```diff
@@ -110,6 +111,9 @@ public static MqttContext fromPublishPacket(
                     .ifPresentOrElse(propertyBag -> {
                         result.topic = ResourceIdentifier.fromString(propertyBag.topicWithoutPropertyBag());
                         result.propertyBag = propertyBag;
+                        // sets the content type using the corresponding value from the property bag
+                        Optional.ofNullable(propertyBag.getProperty(MessageHelper.SYS_PROPERTY_CONTENT_TYPE))
```

Comment:
What if I specify the content type using `Content-Type`?

Reply:
Good point. Field names in [HTTP headers ](https://tools.ietf.org/html/rfc2616) are case-insensitive. I think it makes sense also to treat the property names as case-insensitive in the MQTT _property-bag_.  WDYT?

Reply:
Now I have made it case insensitive and pushed the changes.